addons/binding/org.openhab.binding.keba/src/main/java/org/openhab/binding/keba/handler/KeContactBroadcastListener.java

```diff
@@ -0,0 +1,196 @@
+package org.openhab.binding.keba.handler;
+
+import java.io.IOException;
+import java.net.InetSocketAddress;
+import java.nio.ByteBuffer;
+import java.nio.channels.ClosedChannelException;
+import java.nio.channels.DatagramChannel;
+import java.nio.channels.SelectionKey;
+import java.nio.channels.Selector;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Iterator;
+import java.util.Set;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.ScheduledFuture;
+import java.util.concurrent.TimeUnit;
+
+import org.eclipse.smarthome.core.common.ThreadPoolManager;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+class KeContactBroadcastListener {
+
+    private static final String THING_HANDLER_THREADPOOL_NAME = "thingHandler";
+    public static final int LISTENER_PORT_NUMBER = 7090;
+    public static final int LISTENING_INTERVAL = 100;
+
+    private DatagramChannel broadcastChannel = null;
```

Comment:
These `null`s are also the defaults just as `false` since you already leave them out at line 30 en 31 lets leave out the other ones to.